app/views/admin/world_location_news/edit.html.erb

```diff
@@ -0,0 +1,30 @@
+<% page_title "Edit " + @world_location_news.name %>
+
+<div class="row">
+  <section class="col-md-8">
+    <h1>Edit world location</h1>
+  <p class="warning">Warning: changes to worldwide locations appear instantly on the live site.</p>
```

Comment:
Should this be "changes to world location news articles appear instantly...." or do we not care about publishers knowing the difference?

Reply:
We've kept [the previous wording](https://github.com/alphagov/whitehall/blob/26f54ba8ac13ef5c2da8ba4c14a6dd1b610ded0d/app/views/admin/world_locations/edit.html.erb#L6) as the content being published is exactly the same as before.

The whole interface around world locations probably needs revisiting from a design perspective, so maybe we could ask Publishing Experience to put this on their backlog?